src/YOUR_NAME_ex/ex03/tests/test_median.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from code import median
+
+
+def test_median_for_one_element_list():
+    assert median([1])
+
+
+def test_median_for_odd_numbers():
+    odd_data = [1, 3, 5, 9]
+    assert median(odd_data)
+
+
+def test_median_for_even_numbers():
+    even_data = [2, 4, 6, 8, 10]
+    assert median(even_data)
+
+
+def test_median_multiple_elements():
+    data = {
+        "#1": [1, 2, 3, 5, 8],
+        "#2": [10, 40, 5, 90],
+        "#3": [5.6, 3.8, 2.7, 2.4, 65.22, 0.224],
+    }
+
+    for key in data:
+        multiple_data = median(data[key])
+        assert multiple_data == median(data[key])
```

Comment:
Here you are asserting that the median function returns the same value two times in a row given the same data.